ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -51022,6 +51022,12 @@ periodics:
     job-release: "4.13"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-release-master-nightly-4.13-e2e-aws-ovn-upgrade-rollback-oldest-supported
+  reporter_config:
+    slack:
+      channel: '#ocp-qe-rollback-ota'
+      job_states_to_report:
+      - failure
```

Comment:
Shall we include `error` status here?

Reply:
Good point. Per the [doc](https://pkg.go.dev/k8s.io/test-infra/prow/apis/prowjobs/v1#ProwJobState), 

```
// ErrorState means the job could not schedule (bad config, perhaps).
	ErrorState [ProwJobState](https://pkg.go.dev/k8s.io/test-infra/prow/apis/prowjobs/v1#ProwJobState) = "error"
```

It's fine with me to include it.